cypress/fixtures/valid-execution-with-failure.json

```diff
@@ -7,7 +7,7 @@
}/{substring(file.name, 0, 3)}/test-stack-Rule-1576250154395/\"},{\"regex\":\"^MOD09GQ\\\\.A[\\\\d]{7}\\\\.[\\\\S]{6}\\\\.006\\\\.[\\\\d]{13}_ndvi\\\\.jpg$\",\"sampleFileName\":\"MOD09GQ.A2017025.h21v00.006.2017034065104_ndvi.jpg\",\"bucket\":\"public\",\"url_path\":\"{cmrMetadata.Granule.Collection.ShortName}___{cmrMetadata.Granule.Collection.VersionId}/{substring(file.name, 0, 3)}/test-stack-Rule-1576250154395/\"}],\"reportToEms\":false,\"sampleFileName\":\"MOD09GQ.A2017025.h21v00.006.2017034065104.hdf\",\"name\":\"MOD09GQ_test-test-stack-Rule-1576250154395\",\"granuleId\":\"^MOD09GQ\\\\.A[\\\\d]{7}\\\\.[\\\\S]{6}\\\\.006\\\\.[\\\\d]{13}$\",\"process\":\"modis\",\"dataType\":\"MOD09GQ_test-test-stack-Rule-1576250154395\",\"granuleIdExtraction\":\"(MOD09GQ\\\\..*)(\\\\.hdf|\\\\.cmr|_ndvi\\\\.jpg)\",\"provider_path\":\"cumulus-test-data/pdrs\",\"version\":\"006\",\"updatedAt\":1576250156248,\"duplicateHandling\":\"replace\",\"url_path\":\"{cmrMetadata.Granule.Collection.ShortName}___{cmrMetadata.Granule.Collection.VersionId}/{substring(file.name, 0, 3)}\",\"createdAt\":1576250156248},\"cmr\":{\"username\":\"matthewsavoie\",\"cmrEnvironment\":\"UAT\",\"cmrLimit\":100,\"clientId\":\"cumulus-core-test-stack\",\"oauthProvider\":\"launchpad\",\"provider\":\"CUMULUS\",\"password\":\"CY8t6pT4BPlSV8l3KGSBHcgz075nqMHSWWSrtprp1nTB58SOJexRL0s9nbTgG+uL5tGK5HLiwbYdTJCDgvJLaNiK5/4zPZIUIcBdoeWTRdmxU6v6oPWogGUj1L6FUaih67eZD3XhrHv8xuUEZLPMbrg7whXT5d9UzAd+gbi0/Cc1wTy+fYi2Zgo4FUYgOamm6AFcKYEVtdvyOTxbqp5ZNUockv3FOso/Q8ypNbXFY+tGFMllGhtj5uK1pijKnaqX1dcaOLnV8fQuJYzU2TXHCF97rAektas0Y6b+dNAvPu3um93yByPmeRaZNSkXN+1QL9AsoNwpWU0vltNjV+PSQg==\",\"cmrPageSize\":50},\"workflow_tasks\":{\"0\":{\"version\":\"$LATEST\",\"name\":\"test-stack-HelloWorld\",\"arn\":\"arn:aws:lambda:us-east-1:012345678901:function:test-stack-HelloWorld\"}},\"provider\":{},\"queues\":{\"backgroundProcessing\":\"https://sqs.us-east-1.amazonaws.com/012345678901/test-stack-backgroundProcessing\",\"ScheduleSFDeadLetterQueue\":\"https://sqs.us-east-1.amazonaws.com/012345678901/test-stack-ScheduleSFDeadLetterQueue\",\"startSF\":\"https://sqs.us-east-1.amazonaws.com/012345678901/test-stack-startSF\",\"kinesisFailure\":\"https://sqs.us-east-1.amazonaws.com/012345678901/test-stack-kinesisFailure\",\"triggerLambdaFailure\":\"https://sqs.us-east-1.amazonaws.com/012345678901/test-stack-triggerLambdaFailure\"},\"distribution_endpoint\":\"https://b8f8ybopuh.execute-api.us-east-1.amazonaws.com:7003/DEV/\",\"template\":\"s3://test-stack-internal/test-stack/workflow_template.json\",\"triggerRule\":\"Updated_1576250154395\",\"queueExecutionLimits\":{\"backgroundProcessing\":5}},\"cumulus_meta\":{\"message_source\":\"sfn\",\"execution_name\":\"8e21ca0f-79d3-4782-8247-cacd42a595ea\",\"state_machine\":\"arn:aws:states:us-east-1:012345678901:stateMachine:test-stack-HelloWorldWorkflow\",\"workflow_start_time\":null,\"system_bucket\":\"test-stack-internal\",\"queueName\":\"startSF\"},\"payload\":{\"hello\":\"Hello World\"}}"
```

Comment:
these lines suck to review.  The changes are to the subkey workflow_tasks and just numbering the tasks, rather than having them keyed by name.